process/sandbox.md

```diff
@@ -0,0 +1,99 @@
+
+When we initially created the Inception project level, it was intended to provide an avenue for technically interesting early-stage projects that were beneficial to the cloud-native community.  We are transitioning Inception projects to the Sandbox. Sandbox projects should be early-stage projects that the CNCF TOC believes warrant experimentation. The Sandbox should provide a beneficial, neutral home for such projects, in order to foster collaborative development. We aspire to make the Sandbox the preferred path for early-stage projects to enter the CNCF. More mature projects can continue to jump directly to incubation, but as the cloud-native ecosystem grows, we expect to see proportionally more early-stage projects.
+
+## Early Stage
+
+When we say that Sandbox projects are "early stage" this covers the following examples:
+
+1. New projects that are designed to extend one or more CNCF projects with functionality or interoperability libraries. In the case of Kubernetes, the Sandbox is intended as a home for projects that would previously have [started in the Kubernetes Incubator](https://github.com/kubernetes/community/blob/master/incubator.md).
+2. Independent projects that fit the CNCF mission and provide potential for a novel approach to existing functional areas (or are an attempt to meet an unfulfilled need)
+3. Projects commissioned or sanctioned by the CNCF, including initial code for CNCF WG collaborations, and "experimental" projects
+4. Any project that realistically intends to join CNCF Incubation in future and wishes to lay the foundations for that
+
+## Roadmap for Sandbox Projects
+
+![CNCF Sandbox Roadmap](sandbox.png)
+
+## Caveat Utilitor
+
+The CNCF hopes that all early stage projects will achieve the success they desire.  And the organisation will help as appropriate.  But certain caveats must be stated nonetheless.
+
+End users should treat early stage projects with care.  It is expected that some Sandbox projects may fail.  They may never move to the next maturity level. While many early projects are safe to try out, users must exercise their own judgment.  Some projects may be alpha quality software.  There is no guarantee of production readiness, users, or professional level support.  Where projects enjoy the public support of one or more professional software organisations, those may be seed stage. In short: The CNCF Operating Principle about "no kingmakers" is of special importance in the Sandbox.  
+
+## Sandbox Governance and Benefits
+
+### Advantages of Sandbox vs non-Sandbox for new projects
+
+CNCF will remain fair and open to all projects no matter what their initial provenance.  Should a project apply for CNCF Incubation, the TOC will use the same criteria regardless of origin. This means the TOC will not discriminate in favour of Sandbox projects vs. non-Sandbox projects.  
+
+Therefore the advantages of being in the Sandbox apply prior to application for Incubation, ie.:
```

Comment:
word feels off to me, perhaps:`  .... prior for Incubation are:`  ?
Or maybe its just me ;-)